Convert About section to RST (#245)
mautic/mautic-community-handbook: docs/about/how_can_i_try_mautic.rst

> How can I try Mautic?

Reviewer: ```suggestion
How to try Mautic?
```

> #####################

Reviewer: ```suggestion
##################
```

> Mautic is available to download for free [:xref:`Download Mautic`], allowing you to explore its full capabilities without restrictions when you host it yourself.

Reviewer: ```suggestion
Mautic is available to download for free, just head over to the website where you can :xref:`Download Mautic`, allowing you to explore its full capabilities without restrictions when you host it yourself.
```

If you check the link file for this link, it outputs the words 'download Mautic' so in this context it doesn't really make sense how it's written.

> For optimal performance, we recommend hosting Mautic on a Virtual Private Server (VPS) rather than shared hosting, particularly if you intend to use it in a production setting on lower end hosting which often have limited resources, which may impact Mautic's efficiency and performance.

Reviewer: ```suggestion
For optimal performance, it's recommended to host Mautic on a Virtual Private Server (VPS) rather than shared hosting, particularly if you intend to use it in a production setting on lower end hosting which often have limited resources, which may impact Mautic's efficiency and performance.
```
